users/models.py

```diff
@@ -148,6 +156,10 @@ def full_legal_name(self) -> str:
             return " ".join([self.first_name, self.last_name])
         return ""
 
+    @property
+    def preferred_name(self) -> str:
+        return self.preferred_first_name if self.preferred_first_name else self.first_name
```

Comment:
This would be a good, super quick thing to add tests for in `users/tests/test_models.py`!  It could also be independent of the database, which would make the tests run particularly fast.